experimental/apps-mcp/lib/providers/doc.go

```diff
@@ -0,0 +1,20 @@
+/*
+Package providers contains MCP tool providers.
+
+Each provider implements a set of related tools:
+
+- databricks: Databricks API integration
+- io: Project scaffolding and validation
```

Comment:
Non-blocking: the `io` package (mostly?) provides tools that most third-party agents already have; these should likely be removed.